src/niivue.js

```diff
@@ -158,8 +158,8 @@ export const Niivue = function (opts = {}) {
   this.otherNV = null; // another niivue instance that we wish to sync postion with
   this.volumeObject3D = null;
   this.clipPlaneObject3D = null;
-  this.crosshairPosition$ = new Subject();
-  this.intensityRange$ = new Subject();
+  this.intensityRange$ = new Subject(); // needs to be updated to have an intensity range for each loaded image #172
+  this.scene.location = new Subject(); // object with properties: {mm: [N N N], vox: [N N N], frac: [N N N]}
```

Comment:
I was wondering why this is not named this.scene.location$.

Reply:
Hi @cdrake , The $ convention is new to me. But since it's the known convention in rxjs I will add it back it. It does visually help identify that the property is an observable, so it's a good idea to use the $. I'll update and merge in another PR soon.